### PR TITLE
WIP - feat: Use Google Analytics to better understand end-users

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: jorturfer <jorge_turrado@hotmail.es>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes [#594]

### CODE REVIEW

**Comment by tomkerkhove** [comment]
FYI @chalin - We are adding Google Analytics

**Comment by JorTurFer** [comment]
> FYI @chalin - We are adding Google Analytics

At least I'm trying it

File: layouts/partials/templates/cookie-consent.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            }
+            var script = document.createElement('script');
+            script.onload = callback;
+            script.src = scriptSrc;
+            document.head.appendChild(script);
+        }
+
+        /* This is the part where you call the above defined function and "calls back" 
+        your code which gets executed after the script has loaded */
+        //REPLACE WITH YOUR GA TAG!
```
Comment (arschles) [inline layouts/partials/templates/cookie-consent.html:43]: should we delete this? it appears done

```suggestion
```

File: layouts/partials/templates/cookie-consent.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        loadScriptAsync('https://www.googletagmanager.com/gtag/js?id=G-W43Z7RYFEW', function () {
+            window.dataLayer = window.dataLayer || [];
+            function gtag() { dataLayer.push(arguments); }
+            gtag('js', new Date());
+            gtag('config', 'G-W43Z7RYFEW', { 'anonymize_ip': true }); //REPLACE WITH YOUR GA TAG!
```
Comment (arschles) [inline layouts/partials/templates/cookie-consent.html:48]: same here RE removing the comment

```suggestion
            gtag('config', 'G-W43Z7RYFEW', { 'anonymize_ip': true });
```

**Review by arschles — COMMENTED** [review]
nothing crucial here. just some comments

**Comment by nate-double-u** [comment]
As per @chalin on the [CNCF Service Desk ticket 1008](https://cncfservicedesk.atlassian.net/browse/CNCFSD-1008?focusedCommentId=19231), there is a new GA tag: `G-YSWMV22VTQ`

**Review by chalin — COMMENTED** [review]
I'd suggest dropping the `googleAnalytics` param -- especially since doesn't seem to be used.

**Comment by chalin** [comment]
I just make a quick pass over the PR. I'll take a closer look later if you can get the build to pass.

**Comment by JorTurFer** [comment]
Honestly, I started this but I have no idea about how to do it and I didn't have enough time to figure it out, so if you know how to proceed, please ignore this pr and go ahead

**Pull request closed without merging**